### PR TITLE
G-ChatSaver

### PR DESCRIPTION
**Pull request by iamyarkin**

### CODE REVIEW

File: store/extensions/GChatSaver/extension.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "title": "GChat-Saver",
```
Comment (WiredSpast) [inline store/extensions/GChatSaver/extension.json]: Make sure the name of the folder and the title of the extension are the same, so add a dash to the name of the folder or remove it here

**Review by WiredSpast — CHANGES_REQUESTED** [review]

**Comment by WiredSpast** [comment]
Your icon appears to be rectangular, it will be formatted to a square in the store so it might look a bit stretched

**Comment by WiredSpast** [comment]
The screenshot has to be named screenshot.png

**Comment by iamyarkin** [comment]
done

File: store/extensions/GChatSaver/extension.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "version": "1.0",
+    "categories": ["Fun", "Others"],
+    
+    "source": "https://github.com/iamyarkin/GChat-Saver",
+    
+    "stable": true,
+    
+    "framework": {
+      "name": "G-Python",
+      "version": "1.0"
```
Comment (WiredSpast) [inline store/extensions/GChatSaver/extension.json]: The current version of G-Python is 0.1.6
```suggestion
      "version": "0.1.6"
```

**Review by WiredSpast — CHANGES_REQUESTED** [review]

**Comment by WiredSpast** [comment]
Apparently the screenshot is too big 
image

**Comment by iamyarkin** [comment]
is it done

**Pull request merged**